### PR TITLE
bug/Documents, Profile and Pagination fixes

### PR DESCRIPTION
**Pull request by SepidehShahbazi**
Issues #69, #73 and #75 are completed:

Documents page:

- Removing "Upload new document" button.
- The whole rectangular section for "Drag and Drop Box" is clickable.
- All the icons in the document list are center.
- fixing padding and height styles for the box of the drag and drop component.

Profile Page:

- the layout is like Figma now.

Pagination component:

- Exporting icons from Figma.
- Adding them as SVG files.
- Converting the SVG files to React Components called ArrowForwardIcon.tsx and ArrowBackIcon.tsx with "1px" stroke by default.
- Updating the Paginator.tsx.

Figma: 
FigmaArrowBackIcon

Designed: 
DesignedArrowBackIcon

Documents

Profile

### CODE REVIEW

File: Client/src/app/profile/page.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default function ProfilePage() {
 				width: '100%',
 				margin: '0 auto',
 				minHeight: '80vh',
-				paddingRight: '8rem',
+				paddingRight: '23rem',
```
Comment (mahid797) [inline Client/src/app/profile/page.tsx:11]: This is a simpler fix. Just remove the Box and use MUI Container i.e. 

```
<Container>
     <ProfileClient />
</Container>
```

File: Client/src/app/documents/page.tsx [hunk 2]
```diff
@@ -85,16 +85,10 @@ export default function DocumentsPage() {
...
-							Upload new document
-						</Button>
 					</Box>
 					{/* Drag-and-Drop Section */}
 					<Box mb={5} width="100%">
-						<DragAndDropBox
-							height="21vh"
-							text="Drag and drop your document here or click to upload"
-						/>
+						<DragAndDropBox text="Drag and drop your document here or click to upload" />
```
Comment (mahid797) [inline Client/src/app/documents/page.tsx:91]: You've removed the `height` prop from `DragAndDropBox` here, but not in the other instance. For consistency, could you align this with Figma’s requirements? Either remove the `height` prop entirely or set a default value based on Figma (preferably in `rem` or `vh` for responsiveness). I missed this detail when originally setting up the component.

**Review by mahid797 — CHANGES_REQUESTED** [review]
Looks Good mostly  :+1:

**Comment by gorkem-bwl** [comment]
Looks good to me!

**Comment by gorkem-bwl** [comment]
Thanks. The design looks good!

**Review by mahid797 — APPROVED** [review]

**Review by Paxxack — APPROVED** [review]

**Pull request merged**